calc/src/mechanics/gen8.ts

```diff
@@ -791,7 +788,7 @@ function calculateGen8(
     description.attackerAbility = attacker.ability;
   } else if (
     attacker.hasAbility('Gorilla Tactics') &&
-    ['Gmax', 'Dynamax'].indexOf(defender.name) !== -1
+    ['Gmax', 'Dynamax'].indexOf(attacker.name) === -1
```

Comment:
Is the attacker name literally going to be "Gmax" or "Dynamax"? I think this clearly needs two passes, so they can be searched.

Reply:
I agree that we shouldn't use names to track maxing and should instead use a field to track it. What's the state of the Dynamax branch?

Reply:
I have no clue, sorry, in this repo I'm just a rando who codes.

Reply:
I can add a field to dynamax pokemon and update this once that branch is done

Reply:
I made something similar in #372